jme3-core/src/main/java/com/jme3/environment/EnvironmentCamera.java

```diff
@@ -172,8 +172,7 @@ public Void call() throws Exception {
 
     @Override
     public void render(final RenderManager renderManager) {
-
-        if (jobs.isEmpty()) {
+        if (!isBusy()) {
```

Comment:
I would personally prefer to remove negatives where possible,  hence `isIdle()` would be better in my opinion.

Reply:
It's not really about how it's called.  It's usually more about the default state.  Usually, you try to arrange things so that the default is false.  In which case, isBusy() can make sense.  The steady state is that usually isBusy() returns false.

Reply:
I think it'd be okay either way. If the negative is a major concern, I'd prefer to reverse the sense of the `if` statement:
```java
    if (isBusy()) {
       // process jobs
    }
```
instead of
```java
   if (isIdle()) {
     return;
   }
   // process jobs
```

Bear in mind that the motivation of this PR is to add a public method that might be used many times (in applications) not just in this one place.

Reply:
Makes sense, all good👍